### PR TITLE
[#310] Improve option more readable 

### PR DESCRIPTION
**Pull request by ifirmawan**
## Summary

All changes are known and confirmed with Joy

- [x] Uganda
- [x] Nepal
- [x] Ethiopia

### CODE REVIEW

File: backend/source/wai-uganda/forms/01.json [hunk 1]
```diff
@@ -546,13 +546,13 @@
...
-                "Toilet that flush/pour flush to septic tank",
-                "Toilet that flush/pour flush but don't know where / toilet that flush/pour flush to elsewhere",
+                "Flush/Flush/Pour flush to piped sewer system",
+                "Flush/Pour flush to septic tank",
+                "Flush/Pour flush to don't know where",
                 "Pit latrine with slab",
-                "pit lantrine with slab made of wood / pitlantrine slab with woods",
-                "pit lantrine slab with wood and roofed with grass thatch",
-                "apit lantrine covered with wood and roof is open",
+                "Pit lantrine slab with wood",
```
Comment (dedenbangkit) [inline backend/source/wai-uganda/forms/01.json:553]: Latrine

**Review by dedenbangkit — COMMENTED** [review]
There are some typos here, also inconsistency with slash spaces

File: backend/source/wai-nepal/forms/554360198.json [hunk 2]
```diff
@@ -415,10 +415,10 @@
             {
               "id": 567800082,
               "options": [
-                "Toilet that flush/pour flush to piped sewer system",
-                "Toilet that flush/pour flush to septic tank",
-                "Toilet that flush/pour flush to elsewhere",
-                "Toilet that flush/poor flush but don't know where",
+                "Flush/Pour flush to piped sewer system",
```
Comment (dedenbangkit) [inline backend/source/wai-nepal/forms/554360198.json:418]: Flush/Pour to piped sewer system

File: backend/source/wai-nepal/forms/554360198.json [hunk 3]
```diff
@@ -438,10 +438,10 @@
...
```
Comment (dedenbangkit) [inline backend/source/wai-nepal/forms/554360198.json:441]: Flush/Pour to piped sewer system

File: backend/source/wai-nepal/forms/554360198.json [hunk 4]
```diff
@@ -477,10 +477,10 @@
...
```
Comment (dedenbangkit) [inline backend/source/wai-nepal/forms/554360198.json:480]: Flush/Pour to piped sewer system

File: backend/source/wai-nepal/forms/554360198.json [hunk 5]
```diff
@@ -525,10 +525,10 @@
...
```
Comment (dedenbangkit) [inline backend/source/wai-nepal/forms/554360198.json:528]: Flush/Pour to piped sewer system

File: backend/source/wai-nepal/forms/557950127.json [hunk 6]
```diff
@@ -205,12 +205,12 @@
               "id": 552050126,
               "options": [
                 "Piped supply inside the building /Piped water into dwelling (into the building)",
```
Comment (dedenbangkit) [inline backend/source/wai-nepal/forms/557950127.json:207]: Inconsistent space

File: backend/source/wai-uganda/category.json [hunk 7]
```diff
@@ -465,11 +465,11 @@
...
                         "text": "Type of Toilets",
                         "options": [
-                            "Ventilated Improved Pit latrine (VIP) / vip latrine",
-                            "Ecosan Toilet /ecosan toilet (uddt)/ ecosan toilet uddt",
+                            "Ventilated Improved Pit latrine (VIP) / VIP latrine",
+                            "Ecosan Toilet",
                             "Pit latrine with slab",
                             "traditional pit latrine",
-                            "pour flush toilet"
+                            "Flush/Pour flush toilet"
```
Comment (dedenbangkit) [inline backend/source/wai-uganda/category.json:472]: Flush/Pour toilet

File: backend/source/wai-uganda/forms/01.json [hunk 8]
```diff
@@ -348,31 +348,31 @@
           "required": true,
           "options": [
             {
-              "name": "Toilet that flush/pour flush to piped sewer system",
+              "name": "Flush/Pour flush to piped sewer system",
```
Comment (dedenbangkit) [inline backend/source/wai-uganda/forms/01.json:351]: Flush/Pour to piped sewer system

**Review by dedenbangkit — CHANGES_REQUESTED** [review]
Most of it are okay, but there are options with extra "flush". Example:
Flush/Pour flush to piped sewer system" should be "Flush/Pour to piped sewer system"

File: backend/source/wai-nepal/forms/557950127.json [hunk 9]
```diff
@@ -204,13 +204,13 @@
...
-                "Piped supply outside the building/ Public tap/standpipe",
-                "Tube well / Borehole /Shallow tubewell/borehole",
+                "Piped supply inside the building / Piped water into dwelling (into the building)",
+                "Piped supply outside the building / Public tap/standpipe",
+                "Tube well / Borehole / Shallow tubewell/borehole",
                 "Protected dug well",
                 "Protected spring",
                 "Bottled water",
-                "Rain water",
+                "Rainwater",
```
Comment (dedenbangkit) [inline backend/source/wai-nepal/forms/557950127.json:213]: do we have code here? is this list working? I noticed that the other question has list of dict, but this one is list of string

**Review by dedenbangkit — CHANGES_REQUESTED** [review]
@ifirmawan are the options work with seeder? I see different type of options here.
seeder:

https://github.com/akvo/wai-sdg-portal/blob/3b63ba7b0a9d66e5c4f27563cc8d4dd1c1ffb39a/backend/seeder/form.py#L74

File: backend/source/wai-nepal/forms/557950127.json (hunk 9 above)
Comment (ifirmawan) [reply backend/source/wai-nepal/forms/557950127.json:213]: Hi mas @dedenbangkit, those are list options of dependency, not the question so its works without adding code and as a list of strings.

**Review by dedenbangkit — APPROVED** [review]

**Pull request merged**